scripts/bundle-rhmi-operators.sh

```diff
@@ -79,15 +87,22 @@ generate_bundles() {
 
   for VERSION in $(echo $VERSIONS | sed "s/,/ /g")
   do
-    cd ./$VERSION
-    opm alpha bundle generate -d . --channels $CHANNEL \
+    if [ BUNDLE_ONLY ]; then
```

Comment:
You may want to change this if statement to keep it consistent with the rest of the file. 
```suggestion
    if [ "$BUNDLE_ONLY" = true ]; then
```

Reply:
Thank you